src/app/forms/components/field-error-message/field-error-message.component.ts

```diff
@@ -8,7 +8,7 @@ import { FormNodeWidth } from '@forms/services/dynamic-form.types';
 })
 export class FieldErrorMessageComponent {
   @Input() name: string = '';
-  @Input() error?: string;
+  @Input() error?: string | null;
```

Comment:
nitpick: you probably don't need to change this since we're not using the async pipe anymore

Reply:
It won't compile if you remove this, I think its because warning that uses this input can be null